import-export-cli/cmd/set.go

```diff
@@ -63,36 +63,64 @@ var SetCmd = &cobra.Command{
 func executeSetCmd(mainConfigFilePath, exportDirectory string) {
 	// read the existing config vars
 	configVars := utils.GetMainConfigFromFile(mainConfigFilePath)
+	//Change Http Request timeout
 	if flagHttpRequestTimeout > 0 {
+		//Check whether the provided Http time out value is not equal to default value
+		if flagHttpRequestTimeout != configVars.Config.HttpRequestTimeout {
+			fmt.Println("Http Request Timout is set to : ", flagHttpRequestTimeout)
```

Comment:
Shall we correct the spelling error in "Timout"?

Reply:
Fixed with c672258dbfed4a533e8a19dc8f1ef5050d4bd02e